lib/host-sp-messages/src/lib.rs

```diff
@@ -175,6 +186,181 @@ pub enum KeyLookupResult {
     MaxResponseLenTooShort,
 }
 
+/// Results for an inventory data request
+///
+/// These **cannot be reordered**; the host and SP must agree on them.
```

Comment:
Maybe worth adding a unit test for the serialized values? E.g., `host_to_sp_command_values()` in this file.

Reply:
Done in 6cf4d21a8969e51f5c56862b00e4b6af6c13f2e3